compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/diagnostics/DiagnosticErrorCode.java

```diff
@@ -127,6 +127,115 @@
     ERROR_MISSING_SEMICOLON_IN_XML_REFERENCE("BCE1010"),
     ERROR_INVALID_ENTITY_REFERENCE_NAME_START("BCE1011"),
     ERROR_DOUBLE_HYPHEN_NOT_ALLOWED_WITHIN_XML_COMMENT("BCE1012"),
+
+    // newly added
+
+    // Keywords
+    ERROR_MISSING_PUBLIC_KEYWORD("BCE02001"),
```

Comment:
Shall we merge these with existing groups?
Might have to update some of them though